lib/addons/amp/index.ts

```diff
@@ -116,23 +113,11 @@ export class AmpAddOn implements ClusterAddOn {
             }
             cfnWorkspace = new aps.CfnWorkspace(cluster.stack, 'MyAMPWorkspace', cfnWorkspaceProps);/* all optional props */ 
             const URLEndpoint = cfnWorkspace.attrPrometheusEndpoint;
```

Comment:
Since we are addressing tech debt:

all variables are lower camel case. regardless of the acronyms. const is just a shortcut for readonly variables.